app.rb

```diff
@@ -59,5 +57,8 @@ class AbodenB < Sinatra::Base
     @listings = Listing.all
     erb :listings
   end
-
+  post "/logout" do
+    session.clear
+    redirect "/"
```

Comment:
Think maybe makes more sense to redirect to the login page rather than the root?

Reply:
Agreed! Have updated